package.json

```diff
@@ -14,9 +14,11 @@
     "build:css-watch": "node-sass --output=app/static/stylesheets --output-style=expanded --source-map true --watch app/static/stylesheets",
     "pretest": "npm run test:lint",
     "test": "npm run test:unit",
+    "test:integration": "jest --testPathPattern=test/integration --coverage=false",
+    "test:unit": "jest --testPathPattern=test/unit --coverage=false",
```

Comment:
the build is failing as it cannot find the coverage file to fix and upload to Sonarcloud.
I believe this is caused by the `--coverage=false` that has been added to the `test:unit` script above

Reply:
Ok, I've removed that now